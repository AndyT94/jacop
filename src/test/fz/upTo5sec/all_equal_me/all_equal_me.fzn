var 0..6: X_INTRODUCED_0;
array [1..4] of var int: x:: output_array([1..4]) = [X_INTRODUCED_0,X_INTRODUCED_0,X_INTRODUCED_0,X_INTRODUCED_0];
solve  satisfy;
