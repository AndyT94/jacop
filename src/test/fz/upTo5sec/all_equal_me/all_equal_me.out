x = array1d(1..4, [0, 0, 0, 0]);
----------